jwst/resample/resample_spec_step.py

```diff
@@ -25,6 +25,23 @@ class ResampleSpecStep(ResampleStep):
     def process(self, input):
         input = datamodels.open(input)
 
+        if isinstance(input, ImageModel):
+            slit_model = datamodels.SlitDataModel()
```

Comment:
I think you might want `datamodels.SlitModel()` here, instead of `SlitDataModel`.

Reply:
Yes, doing some more checking, `SlitDataModel` does not inherit `core.schema`, while `SlitModel` does. So that's why there's no meta from the primary getting copied over by .update().